target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4019-eap2200.dts

```diff
@@ -0,0 +1,282 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "EnGenius EAP2200";
+	compatible = "engenius,eap2200";
+
+	keys {
+			compatible = "gpio-keys";
+
+			wps {
+				label = "wps";
+				gpios = <&tlmm 18 GPIO_ACTIVE_LOW>;
+				linux,code = <KEY_WPS_BUTTON>;
+			};
+	};
+
+	aliases {
+		led-boot = &power;
+		led-failsafe = &power;
+		led-running = &power;
+		led-upgrade = &power;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		power: orange {
+			label = "eap2200:orange:power";
+			gpios = <&tlmm 43 GPIO_ACTIVE_HIGH>;
+		};
+
+		lan1 {
+			label = "eap2200:blue:lan1";
+			gpios = <&tlmm 44 GPIO_ACTIVE_LOW>;
+		};
+
+		lan2 {
+			label = "eap2200:blue:lan2";
+			gpios = <&tlmm 45 GPIO_ACTIVE_LOW>;
+		};
+
+		wlan2g {
+			label = "eap2200:blue:wlan2g";
+			gpios = <&tlmm 46 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		wlan5g {
+			label = "eap2200:yellow:wlan5g";
+			gpios = <&tlmm 47 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy1tpt";
+		};
+
+		wlan5g2 {
+			label = "eap2200:yellow:wlan5g2";
+			gpios = <&tlmm 48 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy2tpt";
+		};
+
+		mode {
+			label = "eap2200:blue:mode";
```

Comment:
(This is more of a FYI. You don't need to change anything here)

There was also a push to unify some of the LED functions. A list of LED functions is available from
https://github.com/torvalds/linux/blob/master/include/dt-bindings/leds/common.h#L35

(From what I can tell, there's sadly no mention of this LED in the "EAP2200 User Guide" from the homepage:
"LED Indicators: LEDs for Power, WAN, 2.4Hz, 5GHz, LAN". But the LED with a strange symbol is clearly visible on the various pictures.)

Reply:
Thanks for comment, no changes.